pages/vi/vi-nation.md

```diff
@@ -22,9 +22,9 @@ Make sure vagrant is running and then click [here](http://localhost:3100) to acc
 
 In [Step 4](vi-planetapps.md#Different_Kinds_of_Updates_to_Your_Community), we listed 3 kinds of updates that you might receive on the community side: updates, publications, and surveys. Another important type of update is the reverse: syncing with the nation. This sends data about your community to the nation.
 
-As you can see from the picture below, click on "Manager".
+As you can see from the picture below, click on "Manager" as shown on the image below.
```

Comment:
There is repeatation which explains to look on picture below. Could you just make it once.
```You can click on "Manager" icon as you can see on the picture below```

Reply:
Done.